packages/actions-global/src/actions/handle-action-error.ts

```diff
@@ -9,15 +9,19 @@ import browserHistory from '@commercetools-frontend/browser-history';
 import showApiErrorNotification from './show-api-error-notification';
 import showUnexpectedErrorNotification from './show-unexpected-error-notification';
 
-type ActionError =
-  | Error
-  | {
-      statusCode: TStatusCode;
-      body: {
-        message: string;
-        errors?: TAppNotificationApiError | TAppNotificationApiError[];
-      };
-    };
+type SdkError = {
+  statusCode: TStatusCode;
+  body: {
+    message: string;
+    errors?: TAppNotificationApiError | TAppNotificationApiError[];
+  };
+};
+
+type ActionError = Error | SdkError;
+
+function isSdkError(error: ActionError): error is SdkError {
+  return (error as SdkError).body !== undefined;
```

Comment:
Hm, didn't know that when you're "inside" a type guard we still need to cast the thing.